workflow/kfdrc_RNAseq_workflow.cwl

```diff
@@ -6,7 +6,7 @@ doc: |
   # Kids First RNA-Seq Workflow V4
 
   This is the Kids First RNA-Seq pipeline, which calculates gene and transcript isoform expression, detects fusions ans splice junctions.
-  We are transitioning to to this current version which upgrades several software components.
+  We have transitioned to this current version which upgrades several software components.
```

Comment:
I thought I made that phrasing update already in master...hmmm..

Reply:
Yep. It was in the README but never in the workflow doc.